packit_service/models.py

```diff
@@ -915,7 +956,7 @@ class PullRequestModel(BuildsAndTestsConnector, Base):
     project = relationship("GitProjectModel", back_populates="pull_requests")
 
     job_config_trigger_type = JobConfigTriggerType.pull_request
-    job_trigger_model_type = JobTriggerModelType.pull_request
+    project_event_model_type = ProjectEventModelType.pull_request
 
     @classmethod
     def get_or_create(
```

Comment:
Do we want to pass a `commit_sha` here? (And also when creating a IssueModel, GitBranchModel and ProjectReleaseModel?)